types/vimeo__player/index.d.ts

```diff
@@ -1,7 +1,7 @@
 // Type definitions for @vimeo/player 2.0
-// Project: https: //github.com/vimeo/player.js
-// Definitions by: Denis Yılmaz <https: //github.com/denisyilmaz>, Felix Albert <f.albert.work@icloud.com>
-// Definitions: https: //github.com/DefinitelyTyped/DefinitelyTyped
+// Project: https://github.com/vimeo/player.js
+// Definitions by: Denis Yılmaz <mail@denisyilmaz.de>, Felix Albert <f.albert.work@icloud.com>
```

Comment:
Actually, a github username is better so we can notify you an issues and pull requests related to this package.